build.gradle

```diff
@@ -13,17 +13,21 @@ import com.android.builder.core.DefaultManifestParser;
 buildscript {
     repositories {
         mavenCentral()
+        jcenter()
     }
 
     dependencies {
         classpath 'com.android.tools.build:gradle:1.5.0'
+        classpath 'com.google.gms:google-services:1.4.0-beta3'
```

Comment:
I just copied this directly from google's tutorial page on setting up google analytics for mobile. Not sure why they're suggesting beta but it appears there isn't actually a non-beta version of 1.4.0: https://jcenter.bintray.com/com/android/tools/build/gradle/. I tried updating to 1.5.0 just now and got a compatibility error that I couldn't solve off-hand, and don't think are worth sinking time into now.